pkg/ddl/util/schema_change_notifier.go

```diff
@@ -28,11 +28,12 @@ import (
 // to retrieve the needed information.
 type SchemaChangeEvent struct {
 	// todo: field and method will be added in the next few pr on demand
```

Comment:
When will we delete this TODO?

Reply:
There're 2 type left. Will delete it in the next PR where we finished handling all types